src/components/Login/Test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+import LoginButton from "./LoginButton";
+
+function Test({
+  user, onSignOutClick, handleChange, signInClick, signUpClick, email, password,
+}) {
+  if (user) {
+    return (
+      <div id="user-info">
+        <div className="user-img" />
+        {user.email}
+        <button onClick={onSignOutClick}>
+          <FontAwesomeIcon icon="sign-out-alt" /> Sign out
+        </button>
+      </div>
+    );
+  }
+  return (
+    <div className="Loginwindow user-info">
+      <div className="loginBars">
+        <input id="emailBar" type="email" value={email} onChange={handleChange} placeholder="Email" />
+        <input id="passBar" type="password" value={password} onChange={handleChange} placeholder="Password" />         
+        <LoginButton onClick={signInClick}>Log in</LoginButton>
+        <button id="signupButton" onClick={signUpClick}>Sign up</button>
```

Comment:
You can wrap the inputs in a `form` with an `onSubmit` event listener, so login works when you press enter